test/TorchSharpTest/TestDisposeScopesPackedSequence.cs

```diff
@@ -1,121 +1,82 @@
 using System.Reflection;
-using TorchSharp;
 using Xunit;
 
-namespace TorchSharpTest;
-
-public class TestDisposeScopesPackedSequence
+namespace TorchSharp
 {
-    [Fact]
-    public void MoveDisposeScope()
+    [Collection("Sequential")]
+    public class TestDisposeScopesPackedSequence
     {
-        var sequences = CreateTestSequences();
-        torch.nn.utils.rnn.PackedSequence packed_sequence;
-        var otherScope = torch.NewDisposeScope();
-        using (torch.NewDisposeScope())
+        [Fact]
+        public void MoveDisposeScope()
         {
-            using (torch.NewDisposeScope())
-            {
-                packed_sequence = torch.nn.utils.rnn.pack_sequence(sequences, enforce_sorted: false);
+            var sequences = CreateTestSequences();
```

Comment:
Indentation issue? Are you using TAB?

Reply:
This one is weird, I am using spaces, and it shows fine in my IDE directly on that commit. I even used the arrow keys to count my spaces, there are 20 just like the one line above it. I want to fix this but have never seen that before.  I've even switched to main and back to this branch with my IDE closed, then opened with a text editor and still get 20 spaces and proper alignment. Any ideas what I need to check?

Reply:
It also looks fine in raw view: https://raw.githubusercontent.com/dotnet/TorchSharp/27bd06b368d6f1c1e6f60bd95b87f0cc753fbe82/test/TorchSharpTest/TestDisposeScopesPackedSequence.cs

Reply:
@NiklasGustafsson I think I see the issue. There isn't one. The lines you are seeing above that are _replaced_ by the less indented line. It is properly spaced. It just looks odd because of the diff display. I'm going to close this one.